pom.xml

```diff
@@ -55,8 +55,8 @@
     <docs.dir>docs</docs.dir>
     <!-- this is here because project.basedir evaluates to null in the script build step -->
     <main.basedir>${project.basedir}</main.basedir>
-    <cdap.version>6.1.0-SNAPSHOT</cdap.version>
-    <hadoop.version>2.3.0</hadoop.version>
+    <cdap.version>6.8.0</cdap.version>
+    <hadoop.version>2.10.2</hadoop.version>
     <hydrator.version>2.3.0-SNAPSHOT</hydrator.version>
```

Comment:
Since we're bumping CDAP version to 6.8, can we also bump `hydrator.version` to 2.10?